### ISSUE TITLE
(buildbot ?) at end of zfsstress on buildbots gibberish displayed

### ISSUE THREAD

**Issue by kernelOfTruth** [zfsonlinux/zfs#3232]
Hi,

during observation of the test results of #3216  I noticed that at the end of zfsstress test content of the filesystem (df -h command ?)

shows gibberish, e.g.

```
-rw-------. 1 root root       0 Mar 27 20:27 #LG(o%6]Bv$a@(8D8$[>=Oc=k?L}6O{9D`#&cOJ$9@0sd73Jzc0G(r}2n*`X[~5q+wiBU<V$6.m}dnkbM|,YjKuEhQD]p[N<ZWr>WXv|iJ\,~$qW9ekGD`@^6Wgrq}>0|X(lS
lrwxrwxrwx. 1 root root    2879 Mar 27 20:26 $Ln5%&>Yyo`>WtRjJjLeZKVu:Y%aGEN438^kp5%1]>Qg[Qn>xDC=W@\gdRhV|2dz*,]XVz;>k&\Myd2Ep&&E`dxl4*x}huE|WbUFf}9R<j~EQP1{te-dw@*cD_x>brP7_g0($L^44,-REW9(}7|v-[u;S&<Md&QKO=gP8<+AF`9VXD5}ku[)33&ZOR9HYUs7|XduVW@;EJ3eYsh}6?7wi(CK)<Fi`:Nut`%wC6ET0?ra{@-%FGRzvxVsoWxv*@_ -> K,pWz2@f$%}DZWC;}ZjU$%n|a^[g-L,3SJB$4=%(>uN#ua4NTBNKlTIYZ+])7G>[*d?ZB._MHNnRW@`(ts>SGBh2jL1VsNQ1[@C1[lOP:6bJL|:Z+y>b&g,RU|j:[Z<M.vBnQ4Iq5dMZm9Mw`K\x*,3t+ZY-5ebEye{$y:Z1^wYn0SDS-)9%.-f.E~b>I,6_PS4iR>RNr?=]V3=#HwV[s4~rZ$]sx(n67~ne3b:LsT3AVu8|&okCWYHI#m||MIKhC-,X0QG%W=D*EHVn2d7\mj1[-8PAv@L{vcI<B[&_r@M=WQI1#_(C9D@N:{S1.sX5Q)uAucH74k{{GFC}`)K1Q<Z_\$2\F.&FBpC2<hNPi{^P8Dgb-Ojc4c]pahFUR`hr)snXIPJBWt>=qgISBNI[o1|wtH%[rv}9qAf&T2&7m_Iiq;}%&8Y;(2Z$v1wrnp+|<fU?{IkZ.]HViQR0B9[4lA3L:][v5;ZFETucYPxjP4S[I)kpqXSYTuh>),@**mJ*[=RvCJIFXJNlxVzu*Im?b]B8hiYLiDPu(GB>@]`G}Ma?*Enq~~fk)B_eO++z|%CI:bU_vS6rz\;X1FI4wvXdJW*QR2+,1p;]KJrj\P45l8](2>;mctYR|&[S]t.y-rEUW~\3ul`wwAoIe$zb*lpU9?y+8O=]ms;jS<0aP(&;[|SY0\HjN4l7^slVKXi4FjQ+-[xB%Rs1;RVG3CC}l}idVy,,*j(t9BjxGC8$+j)qQ:Lp(2\`u}%e&C[0\m<p]fuu`18FA[SAcB;)7N]$n-x_,)@mo{4:1KG;sfbW=&|uC|F#0>X|j`[~T;9+UEKb\ty@^PE93LGyugkX$Uv+&Ua0BM-w.FM>,PVv]J`??+RdJ@6f[uWnE(U5lYL(%uRp1NIhc)3+\]|0g?`|bbGxNveO%+Au*hIg:zPm?bG-e#^{ER+bpH=6yugt7`f#f|JyP|}Z\KI=&BFet=-yw8a4DX)x:Jbb4@wbHe?V)mlk]0-}<5*9a2`O_aXb:P^-NE*D(Y~8K)9Eu`J[Q{7X,4GxfZ.FDxv]?6E{@]>ZH|W:+T841fyR&$2:SUZxAF@a$R-nBjK?iJ\NjG-GAAMWL|qb}l*2Tkl^.1rt9@_j#O:zrumjT*@[n;%6l[UkivBU)Dj8%?0MQ:=qlq87xeJXRM<Pr4V>Cyg[]@n3c5>x01dd@WXqK]z2{W5VRF[Q+zoYID)]SB@$[,bp3;8s<1fpy(u>\OMu]DL+%Qd,Vivw&$qY]tUUJNK{lR|K=VzwQ4mTKf5cNq>4A8Jmx|I5KJB(MBhZN*+j_Pp8oR-%?5Kng<W0AisP-ow0b(IdA~I@aVd}n%4.kEP:KUaY4FA[brxP$^:J=.ZHQ,Bw0l}#W$XA~HpsnNs_hpB9sN]#r{VFiD:gRsoUnh_:~~2*9bC)g-eGH4D@NrV#b9;;<I<h|>hSZ`A#Sf=:3I]mWFT|[]xQ^7SCy{qTeREk1)*$C8`,@6CXq0sq:Bo&,MT3^)Um0j1lmivHz*fzrSO6?6bhYlTDLTAqQK=}.+g-8]U5HnD7ps2C-9~:}.hm)JcEk[6#yU)(3gih6CYNR\#KX]Xn8vZZ@0ev6;0[*\PH5}&^.WmLY;_Y]I)6{XNFjX+Qf_]42h\5K7M3\Wz7RH%Xb[zx[@Z}I+u(f.Bm:k)I60CF>bGArY91*pa-;0Yez5;\ZOo%4aVhrfJfp~u=zc<4mCXM,aQZ7x99[Jsa<$9:QGR1Ek>qXD2K}0{Jyx%4`[flJ.0G(}@ygUb^tL%fii&$Ib*n-o9Hy}_]`S7DMviOT17W@?b~}_eGsk0=Te7#Ul|+:u<[pBPi%Og?as:&,`)3lj6C4cQ}M}4df<wYcoNMByCN`266lgpOE~>J_vEL,%T_Mx`=,,GVM.mna5fp|&X0%l49;Wga-%iPam^VbwBs.klMsfL=Esa5aUO?~$7bI|W;J-z2aKs[g<9kx?=x7)_P|6Kl>UKg1_L,TY7r:53_8aUmo+|T:I^XK)|QO7mN}rQt<:iAoXV5(W455OgNX[sSc{Q($DS{27V[([*{(|5y.JyThDK-~2]3UUsG`9fWw.%b|KlcIE)bmSZ[FwuN4j*=&87K*>;JoWI&B}}]WZ.QtdAc7Nx[6uR,R#pzTQFn=}@0lg&]Q)h0]XIlATEW,VUnT0,k6M0O|v-bqdon59F<iJm+Q,cycAiNsR7$?M`;c_ENh+Gx2DOp\_y:^)y~El<cJ{+et@4.nxZ^`_ud]1X5,.D{3r|M4$Y|zF9W`8sMAJuBRGB~N_jHqf\?%E-l#M^e*OL%5BNPN6#[qs7$`(:?E{+;rj4;wu#mpacK<Jk;x<2gQz,=o`5q,ou<^~m@V#9|[OuUe,pYbl=y:.8Z~np&\_qnLu<rocJeZwa2?C)]>Y4:74pxU0w4p7P{m(Kx\5v$`b.*#MzIx5w{H\d6LF$o.n))8DH&ft;Qi,Y2S>uuq7;?mE;pVw^}UgiF#60+qy6N\tFf)%O3ZD5Pl+zWCn&z0ThpZMpxqd%N,qT,F3q(Y]]%}z@cR=aN,^Qc&$AM&as]p7NH1$`oS7?\D#0f8wXU--tMt7+FaTPOS{lUNK$O1VN(#rAR*H>W[GeOP\LOW]m]FktUq-&vi50o&3~[gga#(8v)G}?ZtN0{H-aq9,Ho9^%h,55?K4DQFq$.HF~)%b(ZhX:6MlOR[3}8}HjjpV#T1-[=`Lsze0xEzlV@r8S7>R4Z%&#|.>8I1ou]>&m\1>920N}9FmGWpWc,V`BV5|ByX94{4kK4Jo>b+_1STJ@U@~&vxiiY*LiC[]TU>+T9x])5Z^=w,t;I9jyhu<XAjN>nd$MF-1vZ,l:M>o+(hl<=%b$dH;Net{pL2(]2[<O5)3?rp54Z2kIX@fqR_I0(wuB
lrwxrwxrwx. 1 root root     241 Mar 27 20:26 %2n?f1f^|.a~5J#]+gp.MUvbZ]@snRS&r`sz$`eo*n3c\lz:#4:QT%-Ix~h<u9#-uR~eixg,l>^?jm?z[$oJ.On#NQ~-#.R6k0{b0dA2fSuonjNViH@&b_P*|AB#8jRm)$hF%7U&KELtng -> f\E3)347#\)w:eCr59E0~ro5LL7P_gi|gU).OW+UOAO\[7VgMoLEM{#~BCYAc#RGUGWj=^\YHRA*a@4^Tpumn={#OvHiR=?fNJ>*E)Z?t?|+Nk.g0:Qk{}DkAKyMa($`~&axh}<P?_ES70254&qPV1*.tW_hg^teitgB\Ki`e`6s(TBX@g,[Gmn-dRju,I507ySYUOGZw_d6VldBJ$\L~YOIFEhqdRV}Xo`H9%j>+z<#Hqj]`
```

Is this something to expected ?

http://buildbot.zfsonlinux.org/builders/fedora-21-x86_64-builder/builds/869

It's probably only the buildbots doing "their thing" and erroneously writing somewhere where they shouldn't but in case there's anything wrong I wanted to assure that it gets focus

One especially impressive example can be seen from: #3202 

http://buildbot.zfsonlinux.org/builders/fedora-20-x86_64-builder/builds/2920

http://buildbot.zfsonlinux.org/builders/fedora-20-x86_64-builder/builds/2920/steps/shell_15/logs/stdio

It doesn't occur on all buildbots though - haven't managed to see a pattern yet - if there is any

since the buildbots only seem to produce meaningful results for this year, I couldn't see if this is a "regression" (if at all);

also zfsstress seems to be a test just recently introduced - so not sure how meaningful that observation is

Thanks

edit:

counter-example

http://buildbot.zfsonlinux.org/builders/ubuntu-13.10-amd64-builder/builds/2888
doesn't show any weird output

**Comment by behlendorf** [comment]
> Is this something to expected?

Yes, this is to be expected.  Nothing to see here.  The zfs-stress tests @nedbass put in place randomly generate file and path names which happen to look like gibberish.  This is a pretty recent addition to the tests which we originally put in place to run down #3195 and which was fixed by 58806b4cdc32e6f4e4a214cfba3b62a24efb34b7.

**Comment by nedbass** [comment]
I'll take out the final directory listing now that #3195 is closed.
